apps/api/src/app/content-templates/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -54,8 +54,14 @@ export class CompileTemplate {
     }
 
     const template = Handlebars.compile(templateContent);
+    let html = null;
+    try {
+      html = template(command.data);
 
-    return template(command.data);
+      return html;
+    } catch (error) {
+      return html;
+    }
```

Comment:
I would actually not return the html here, but throw an ApiError and pass the `handlebars` content as information there. This error could be watched from the calling function in send-message.email level and than create the execution logs with the actual handlebars error.